src/main/webapp/app/overview/course-statistics/course-statistics.component.ts

```diff
@@ -149,14 +160,23 @@ export class CourseStatisticsComponent implements OnInit, OnDestroy, AfterViewIn
     modelingPointLabel = 'modelingPointLabel';
     textPointLabel = 'textPointLabel';
     fileUploadPointLabel = 'fileUploadPointLabel';
+    presentationPointsLabel = 'presentationPointsLabel';
     missingPointsLabel = 'missingPointsLabel';
-    labels = [this.programmingPointLabel, this.quizPointLabel, this.modelingPointLabel, this.textPointLabel, this.fileUploadPointLabel, this.missingPointsLabel];
+    labels = [
+        this.programmingPointLabel,
+        this.quizPointLabel,
+        this.modelingPointLabel,
+        this.textPointLabel,
+        this.fileUploadPointLabel,
+        this.presentationPointsLabel,
+        this.missingPointsLabel,
+    ];
 
     ngxDoughnutColor = {
         name: 'Your overall points color',
         selectable: true,
         group: ScaleType.Ordinal,
-        domain: [], // colors: orange, turquoise, violet, bordeaux, green, red
```

Comment:
I would prefer if you add a type so that it can be type-checked

```ts
type doughnutDomainColor = 'orange' | 'turquoise' | 'violet' | 'bordeaux' | 'green' | 'red'
```